sklearn/neighbors/base.py

```diff
@@ -419,7 +479,10 @@ def radius_neighbors(self, X, radius=None, return_distance=True):
         Parameters
         ----------
         X : array-like, last dimension same as that of fit data
-            The new point or points
+            The new point or points.
+            If not provided, X is assumed to be the same as the fit
```

Comment:
Perhaps, "If not provided, neighbors of each indexed [or: training] point are returned." I'm not sure, then that it needs to explicitly state something like "In this case, the query point is not considered its own neighbor."